scipy/spatial/transform/rotation.py

```diff
@@ -281,3 +282,147 @@ def as_rotvec(self):
             return rotvec[0]
         else:
             return rotvec
+
+    @classmethod
```

Comment:
Do we need it as classmethod? If not, then I suggest to move it out from the class completely.

Reply:
You will make it a standalone function, right?

Reply:
It's not clear from the limited context here but the function just after `as_rotvec` is now `_elementary_quat_compose`. Given that this function accepts information about `seq`, `angles` and the kind of rotation (`intrinsic` / `extrinsic`), does it make sense to move this logic out of the class? Just trying to understand your thought process here.

Reply:
You can mark it as `@staticmethod`, right? But I don't see much value in having static methods, especially non-public ones. It comes down to preferences, but I think keeping the class code shorter is a good thing. Additionally, you already move out some "static methods", why make an exception here. 

I don't insist on changing it, you can mark it staticmethod.